aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -1,6 +1,145 @@
 spec aptos_framework::account {
+    
+    spec module {
+        pragma verify = true;
+        pragma aborts_if_is_strict;
+    }
+
     spec create_signer {
-        // TODO: temporary mockup.
         pragma opaque;
+        aborts_if [abstract] false;
+        ensures [abstract] signer::address_of(result) == addr;
+    }
+
+    spec initialize {
```

Comment:
It know that current specs also not do this yet, but can we please start with providing the parameter lists here? This makes the specifications more readable.

```
spec fun_name(parameters_as_in_definition): result_type {
  ...
}
```